docs/openapi.yml

```diff
@@ -278,6 +278,62 @@ paths:
               schema:
                 $ref: '#/components/schemas/datasets'
 
+  /datasources:
+    get:
+      summary: List all datasources
+      description: Returns a list of datasources.
+      tags:
+        - Datasources
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/datasources'
+    post:
+      summary: Create a datasource
+      description: "Creates a new datasource and returns the created datasource.
```

Comment:
**description**: Creates a new datasource object. A datasource is the physical location of a dataset such as a table in `PostgreSQL`, or topic in `Kafka`. A datasource enables the grouping of physical datasets to their physical source.